### ISSUE TITLE
SQLyog wrongly interprets "CHECK" option in ENUM as a CHECK CONSTRAINT

### ISSUE THREAD

**Issue by H0w4rd** [webyog/sqlyog-community#2312]
You can reproduce this bug if you create a table as following
```
CREATE TABLE `settings` (
  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `key` varchar(100) NOT NULL,
  `value` mediumtext DEFAULT NULL,
  `blob` mediumblob DEFAULT NULL,
  `form_type` enum('TEXT','TEXTAREA','WYSIWYG','DATE','DATETIME','INTEGER','FLOAT','CHECKBOX','ARRAY','PASSWORD','BLOB') DEFAULT NULL COMMENT 'Typ formulářového prvku',
  `settings_id` int(11) unsigned DEFAULT NULL,
  `order` int(10) unsigned DEFAULT 1,
  `not_deleted` tinyint(1) unsigned DEFAULT 1 COMMENT 'Nesmazany',
  PRIMARY KEY (`id`),
  UNIQUE KEY `settings_UK_key` (`key`,`not_deleted`),
  KEY `settings_id` (`settings_id`),
  CONSTRAINT `settings_ibfk_1` FOREIGN KEY (`settings_id`) REFERENCES `settings` (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8
```

Then open it with F6 (Alter table) and you will see this under CHECK CONSTRAINT:
`BOX','ARRAY','PASSWORD','BLOB') DEFAULT NULL `

![image](https://user-images.githubusercontent.com/9164291/211525844-60c906c7-9329-42d6-9b4c-92025bea74a6.png)

This causes troubles later when altering the table, generated SQL command is wrong and has to be manually fixed (CHECK CONSTRAINT part).

Please could you fix it?

Version 13.2.0

**Comment by sibwy** [comment]
This is reproducible in SQLyog 13.2.0 with MySQL v8.0.31 and MariaDB v10.8.3.

**Comment by sibwy** [comment]
Fixed in [SQLyog v13.2.1](https://webyog.com/blog/sqlyog/new-release-sqlyog-13-2-1/)
